pkg/cloud/services/ec2/subnets.go

```diff
@@ -316,7 +316,10 @@ func (s *Service) deleteSubnet(id string) error {
 
 func (s *Service) getSubnetTagParams(id string, public bool) infrav1.BuildParams {
 	var role string
-	var additionalTags infrav1.Tags
+	additionalTags := s.scope.AWSCluster.Spec.AdditionalTags
```

Comment:
If we have a method to retrieve these, we can do the init in one place, on retrieval, wdyt?

Reply:
Are you thinking of something like this?
```go
s.scope.AdditionalTags().Merge(infrav1.Tags{
  infrav1.NameKubernetesAWSCloudProviderPrefix+s.scope.Name()]: string(infrav1.ResourceLifecycleShared),
})

Reply:
Yeah